src/NuGet.Core/NuGet.Packaging/ContentModel/ManagedCodeConventions.cs

```diff
@@ -65,6 +65,8 @@ public class ManagedCodeConventions
                 AnyFramework.Instance )
         });
 
+        private static readonly FrameworkReducer FrameworkReducer = new();
```

Comment:
Should we have a static `Default` on `FrameworkReducer` itself, and look for other places where it's being created?

Reply:
We could consider that. 

It's only really used in a few places in VS. My intention was to scope the change.

Reply:
@zivkan 

Do you care if I merge it as is, or do you want me to look into it further?

There's no other restore codepaths where ManagedCodeConventions is being used.

Reply:
I didn't look into where/how often it's being created, so I'll trust your judgement.